wled00/data/index.js

```
@@ -3306,6 +3306,134 @@ function simplifyUI() {
+}
+
+function savePresetsGen()
+{
+       if (!confirm('Are you sure to (over)write presets.json?')) return;
+
+       uploadFileWithText("/presets.json", gId("presetsGen").value);
+}
+
+function savePresetsGen()
+{
+	if (!confirm('Are you sure to (over)write presets.json?')) return;
```

Biome (1.9.4)

[error] Shouldn't redeclare 'savePresetsGen'. Consider to delete it or rename it.

'savePresetsGen' is defined here:


(lint/suspicious/noRedeclare)